.github/workflows/docs.yml

```diff
@@ -0,0 +1,23 @@
+name: Docs
+
+on:
+  push:
```

Comment:
Is there also a trigger for "on successful build/checks"? That would probably be better than on every push.

Reply:
This should only trigger on pushes to the master branch, so after a PR has been merged. At that point the checks should have already passed. Another option is to rebuild the docs [on releases](https://help.github.com/en/articles/events-that-trigger-workflows#release-event-release), but at least for now I think it's helpful to have the docs track master.

See also the [workflow syntax docs](https://help.github.com/en/articles/workflow-syntax-for-github-actions) and [event descriptions](https://help.github.com/en/articles/events-that-trigger-workflows).